ui_framework/components/info_button/_info_button.scss

```diff
@@ -0,0 +1,7 @@
+.kuiInfoButton {
+  font-size: 16px;
+  cursor: pointer;
+  border: none;
+  background-color: inherit;
+  padding: 0;
+}
```

Comment:
In Safari, I noticed that this icon becomes white when you click on it. How about adding some hover and active states to prevent this?

```scss
.kuiInfoButton {
  /* styles... */
  color: $infoButtonColor;

  &:hover,
  &:active {
    color: $infoButtonColor;
  }
}
```